### PR TITLE
feat: Add links to libraries

### PR DESCRIPTION
**Pull request by stefanjudis**
Here we go. :) @kdamball.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +25,10 @@ npx @contentful/create-contentful-app create-definition
 ```
 
 [Read more](https://www.contentful.com/developers/docs/extensibility/app-framework/create-contentful-app/) and check out the video on how to use the CLI.
+
+## Libraries to use
+
+To make your app look and feel like Contentful use the following libraries:
+
+- [Forma 36](https://f36.contentful.com/) – Contentful's design system
```
Comment (kdamball) [inline README.md:33]: We already have forma at the top :)

**Review by kdamball — COMMENTED** [review]
I would actually include these only in https://github.com/contentful/create-contentful-app/blob/master/template/README.md if possible. Note that this is in the bootstrapped app's README :) wdyt @fabe

File: README.md (hunk 1 above)
Comment (stefanjudis) [reply README.md:33]: 🤦 Sorry about that. :D

File: README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
 # Create Contentful App
 
-This project will make it easy to bootstrap [Contentful Apps](https://www.contentful.com/developers/docs/extensibility/app-framework/), with [React](https://reactjs.org/),
-[Forma 36](https://f36.contentful.com/), and [Field Editors](https://www.contentful.com/developers/docs/extensibility/field-editors/).
+This project makes it easy to bootstrap [Contentful Apps](https://www.contentful.com/developers/docs/extensibility/app-framework/) with [React](https://reactjs.org/) and [Forma 36](https://f36.contentful.com/) (Contentful's design system).
```
Comment (stefanjudis) [inline README.md:3]: I removed the field editors because they're not included by default. :)

**Review by fabe — APPROVED** [review]

**Review by kdamball — COMMENTED** [review]
@stefanjudis can you add a version bump from latest master and also run `npm i` to make sure lock file is also updated?

**Pull request merged**